build.sbt

```diff
@@ -16,10 +16,31 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 ThisBuild / tlSitePublishBranch := Some("main")
 
 val Scala213 = "2.13.8"
-ThisBuild / crossScalaVersions := Seq(Scala213, "3.1.2")
+val Scala3 = "3.1.2"
+
+ThisBuild / crossScalaVersions := Seq(Scala213, Scala3)
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
-lazy val root = tlCrossRootProject.aggregate(core, java)
+ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+ThisBuild / semanticdbEnabled := !tlIsScala3.value
+ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
+ThisBuild / scalafixScalaBinaryVersion := {
+  CrossVersion.binaryScalaVersion(scalaVersion.value)
+}
+
+ThisBuild / githubWorkflowBuild := {
+  val scalafixStep = WorkflowStep.Sbt(
+    List("root/scalafixAll --check"),
+    name = Some("Check that scalafix has been run"),
+    cond = Some(s"matrix.java == 'temurin@8' && matrix.scala != '$Scala3'")
+  )
```

Comment:
`root/scalafixAll` will run for all modules. `scalafixAll` will run for only the current active project, which in CI is either `rootJVM` or `rootJS`, which is more efficient.